### PR TITLE
Adding Microsoft.Insights/alertRules resource template

### PR DESCRIPTION
**Pull request by nmackenzie**
Added a template that creates a Microsoft.Insights/alertRules resource which sends alerts to service administrators and additional specified email addresses when an Azure Service Health incident is reported for a resource group.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@nmackenzie__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: insights-alertrules-servicehealth/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+There are 3 types of alert:
+* Active
+* InProgress
+* Resolved
+Each of them must be added separately.
+
+It is an ARM template implementation of the [Create or update an alert rule](https://msdn.microsoft.com/en-us/library/azure/dn933805.aspx) operation in the Azure Resource Manager Insights API. The creation of these alerts in C# is described in this [post](https://code.msdn.microsoft.com/How-To-Setup-Email-Alerts-c26cdc55) by Matt Loflin.
+
+Below are the parameters that the template expects:
```
Comment (singhkay) [inline insights-alertrules-servicehealth/README.md]: The table of parameters is not needed as they are rendered already on Azure.com

**Comment by singhkay** [comment]
@nmackenzie Looks good, just one change needed

**Comment by singhkay** [comment]
Removing cla as @nmackenzie is MSFT FTE. merged.

**Comment by singhkay** [comment]
@nmackenzie Realized after the fact that this is a platform capability. Can you rename the folder to 201-insights-alertrules-servicehealth and submit another PR. Thanks!

**Pull request merged**